### PR TITLE
Config screen

### PR DESCRIPTION
**Pull request by lotuuu**
Adds a screen to edit the json config files. Implemented config files are `Characters.json`, `Skills.json` and `GameSettings.json`. Implemented in liveview to handle the addition of new entries.

Characters and skills are listed by name. Name (and id for Characters) are required fields. For Game Settings, all fields are required.

Addition of new characters and skills, as well as custom values for them, has been implemented. Values need a name to be created, while entities don't (they get assigned an indexed name).

Screen should be styled later on.

### CODE REVIEW

File: server/lib/dark_worlds_server/utils/config/config.ex [hunk 1]
```diff
@@ -0,0 +1,42 @@
+defmodule Utils.Config do
+  @streaming_assets_path "../client/Assets/StreamingAssets/"
```
Comment (AminArria) [inline server/lib/dark_worlds_server/utils/config/config.ex:2]: Not to solve here, but better mention it

This will break once we split the repos

**Review by tvillegas98 — DISMISSED** [review]
Nice work! 😄

File: server/lib/dark_worlds_server/utils/config/config.ex (hunk 1 above)
Comment (lotuuu) [reply server/lib/dark_worlds_server/utils/config/config.ex:2]: Yes, this will be fixed in a future PR!

**Review by BertovDev — APPROVED** [review]
Superb !! 🤩

**Pull request closed without merging**